docs/source/actions.rst

```diff
@@ -196,13 +196,19 @@ List of recipients of the message. It could be a channel or an user.
 
 SPIDERMON_SLACK_SENDER_NAME
 ---------------------------
+Name of app/bot/user created for serving notifications on desired channels.
 
 .. _SPIDERMON_SLACK_SENDER_TOKEN:
 
 SPIDERMON_SLACK_SENDER_TOKEN
 ----------------------------
 
-Your `Slack token`_.
```

Comment:
I think the existing line is better than the first line of the replacement text. People that already know what a Slack token is will understand this right away. People who do not can follow the link.

Reply:
Actually, what I felt going new into both Slack API, is that there is nothing called Slack Token. There are several tokens which have different functions and I felt the need to explain that first.